fix: include schema option in route declaration

abahernest opened the pull request:
Include schema definition when declaring route.

I noticed that you added the generated json schema definition to fastify via the method `fastify.addSchema()` 
![image](https://github.com/Coobaha/typed-fastify/assets/32009497/c095310f-0f27-4b27-b85d-c61cad5425cd)

But you didn't add the generated routes schema when you iteratively declared all the routes.
![image](https://github.com/Coobaha/typed-fastify/assets/32009497/0fc60ce8-e6f9-47db-a650-813b8e2bcd6d)

I noticed this while using the library alongside @fastify/swagger and it couldn't generate documentation for the routes that were already declared. 
Adding the generated schema to `fastify.route()`  fixed it for me.

Coobaha commented:
@abahernest Thank you for the contribution. Could you please bring context to the change?

I wonder why the tests are failing in CI. Did you have a chance to run tests locally?

abahernest commented:
> @abahernest Thank you for the contribution. Could you please bring context to the change?
> 
> I wonder why the tests are failing in CI. Did you have a chance to run tests locally?

I've fixed the failing test and added a description to the PR.

Coobaha commented:
Thank you!

abahernest commented:
You're welcome. Glad to help 💪
